libcloud/loadbalancer/drivers/elb.py

```diff
@@ -308,8 +308,7 @@ def ex_create_balancer_listeners(self, name, listeners=None):
             if protocol == 'HTTPS' or protocol == 'SSL':
                 params['Listeners.member.%d.   \
                         SSLCertificateId' % i] = listener[3]
-        else:
-            return False
+        return False
```

Comment:
actually I just noticed 2 bugs I had forgotten to fix so will push these and take that opportunity to revert the `for` `else` change here to avoid confusion